src/keywords.YAML-tmLanguage

```diff
@@ -2,7 +2,7 @@ repository:
     keywords:
         patterns:
             -
-                match: '(\!in|\!is|as\?)'
+                match: '(\!in|\!is|as\?)\b'
```

Comment:
I think we probably want the entire token in the word boundary.
```suggestion
                match: '\b(\!in|\!is|as\?)\b'
```

Reply:
Probably not, check the third test added.

Reply:
Ah I see 🤔